backbone/src/main/java/org/researchstack/backbone/task/OrderedTask.java

```diff
@@ -168,4 +168,9 @@ public void validateParameters() {
     public List<Step> getSteps() {
         return new ArrayList<>(steps);
     }
+
+    @Override
+    public void resetCompletedTask(List<String> list) {
+
```

Comment:
Can you add a comment explaining why this isn't implemented?
E.g.:

``` 
    @Override
    public void resetCompletedTask(List<String> list) {
         // no-op: During a ReviewStep, the user can change its responses. In Ordered tasks, step Ids don't change and they are always in the same position, so there is no need to reset it. 
    }
```

(or whatever better description you can come up with)

Thanks, this will help future developers understand what's going on here.